src/main/java/com/google/sps/data/CareerQuestionDatabase.java

```diff
@@ -22,37 +17,41 @@
   private static final String CHOICETEXT_QUERY_STRING = "choicetext";
   private static final String ASSOCIATED_CAREER_PATH_QUERY_STRING = "associatedcareerpath";
   private static final Query query = new Query(ENTITY_QUERY_STRING);
-  
+
   public CareerQuestionDatabase(DatastoreService datastore) {
     this.datastore = datastore;
   }
 
   /* method to return all questions and choices from database as an arraylist */
   public ArrayList<CareerQuestionAndChoices> getQuestionsAndChoices() {
-    ArrayList<CareerQuestionAndChoices> careerQuestionAndChoicesList = new ArrayList<CareerQuestionAndChoices>();
-    List<Entity> entities = this.datastore.prepare(this.query).asList(FetchOptions.Builder.withDefaults());
+    ArrayList<CareerQuestionAndChoices> careerQuestionAndChoicesList =
+        new ArrayList<CareerQuestionAndChoices>();
+    List<Entity> entities =
+        this.datastore.prepare(this.query).asList(FetchOptions.Builder.withDefaults());
     for (Entity entity : entities) {
-      CareerQuestionAndChoices careerQuestionAndChoices = this.entityToCareerQuestionAndChoices(entity);
+      CareerQuestionAndChoices careerQuestionAndChoices =
+          this.entityToCareerQuestionAndChoices(entity);
       careerQuestionAndChoicesList.add(careerQuestionAndChoices);
     }
     return careerQuestionAndChoicesList;
   }
 
-  public void putCareerQuestionAndChoicesIntoDatabase(CareerQuestionAndChoices careerQuestionAndChoices) {
-      String question = careerQuestionAndChoices.getQuestion();
-      Entity questionEntity = getQuestionDatastoreEntity(careerQuestionAndChoices);
-      this.datastore.put(questionEntity);
-      for (CareerQuestionChoice choice : careerQuestionAndChoices.getChoices()) {
-        Entity choiceEntity = getChoiceDatastoreEntity(question, choice);
-        this.datastore.put(choiceEntity);
-      }
-  } 
+  public void putCareerQuestionAndChoicesIntoDatabase(
+      CareerQuestionAndChoices careerQuestionAndChoices) {
+    String question = careerQuestionAndChoices.getQuestion();
```

Comment:
wonky indents, probably a whitespace goof

Reply:
I think its a 4 space line continuation indent followed by a normal 2 space indent, linter *should* catch whitespace problems.